src/main/java/seedu/address/model/project/UniqueProjectList.java

```diff
@@ -69,6 +69,18 @@ public void setProject(Project target, Project editedProject) {
         internalList.set(index, editedProject);
     }
 
+    // TODO: temporary getter for project, can be deleted when alt method to retrieve project is implemented
+    public static Project getProject(int projectId) {
+        for (Project p : internalList) {
+            ProjectId id = new ProjectId(projectId);
+            if (p.getId() == id) {
```

Comment:
`==` may cause some unforeseen bugs later on, `.equals()` might be a better choice